src/PublicApiGeneratorTests/Event_modifiers.cs

```diff
@@ -0,0 +1,138 @@
+﻿using PublicApiGeneratorTests.Examples;
+using System;
+using Xunit;
+
+namespace PublicApiGeneratorTests
+{
+    public class Event_modifiers : ApiGeneratorTestsBase
```

Comment:
This file is a direct copy of Property_modifiers with a few search and replaces.